tools/api-markdown-documenter/api-report/api-markdown-documenter.alpha.api.md

```diff
@@ -758,23 +764,24 @@ export interface TextFormatting {
     readonly strikethrough?: boolean;
 }
 
-// @alpha
+// @public
 export interface ToHtmlConfig extends ConfigurationBase {
     readonly customTransformations?: ToHtmlTransformations;
     readonly language?: string;
     readonly startingHeadingLevel?: number;
 }
 
-// @alpha
-export interface ToHtmlContext extends ConfigurationBase {
+// @public
+export interface ToHtmlContext {
     readonly headingLevel: number;
+    readonly logger: Logger;
```

Comment:
It was alpha so we can change it, but at least worth calling out in the CHANGELOG?

Reply:
The `@alpha` APIs weren't actually exported at all yet (we don't have an `/alpha` entrypoint in this package). So no one could have been using this yet :)